src/Compilers/CSharp/Portable/Symbols/TypeParameterBounds.cs

```diff
@@ -34,14 +38,20 @@ internal sealed class TypeParameterBounds
             this.Interfaces = interfaces;
             this.EffectiveBaseClass = effectiveBaseClass;
             this.DeducedBaseType = deducedBaseType;
-            this.IgnoresNullableContext = ignoresNullableContext;
+            this.UsedLightweightTypeConstraintBinding = usedLightweightTypeConstraintBinding;
         }
 
         private TypeParameterBounds()
         {
         }
```

Comment:
Consider removing this constructor.

Reply:
It may be silly, but I didn't want to call passing an explicit `usedLightweightTypeConstraintBinding: false` value for the `Unset` constant, given that it has a very different meaning.